### PR TITLE
Add snow tileset and artwork to TS.

### PR DESCRIPTION
**Pull request by pchote**
tssnow

### CODE REVIEW

File: mods/ts/tilesets/temperat.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 General:
 	Name: Temperate
 	Id: TEMPERAT
-	Extensions: .tem, .shp
+	Extensions: # Obsolete
```
Comment (Phrohdoh) [inline mods/ts/tilesets/temperat.yaml]: So remove it?

Comment (pchote) [inline mods/ts/tilesets/temperat.yaml]: That is the next PR.  We need to convert all the mods to this new system first.

**Comment by obrakmann** [comment]
Looks ok to me. The snow terrain looks good afaict, other mods still work, editor too. :+1:

**Comment by reaperrr** [comment]
Tested and looked through yamls, looks good except for one thing:
As mentioned on irc, it doesn't pass ```--check-sequence-sprites``` due to gagreen not having an shp, but that's not a blocker.
:+1:

**Pull request merged**